pkg/timescale_prometheus/main.go

```diff
@@ -0,0 +1,339 @@
+// This file and its contents are licensed under the Apache License 2.0.
```

Comment:
It seems odd that the main (runner) of the repo is in pkg than the actual cmd. Is there any reason or plan more than this? Like, multiple calls to `Run()`?

Reply:
If there are plans, then the design of Run can be better. Will look into that after confirming the answer for the above question.

Reply:
Testing nonsense: we can't use `internal` packages in `cmd` and we can't use binary packages in `end_to_end` tests

Reply:
This file required renaming.